docker/assertj/src/main/java/com/github/dockerjava/assertions/ContainerAssert.java

```diff
@@ -175,4 +196,23 @@ private ContainerConfig getContainerConfig() {
       return this.actual.getConfig();
    }
 
+    private static String getProcessName(int processId) {
```

Comment:
I was thinking in inspect the output of https://docs.docker.com/engine/api/v1.23/#/containers section List processes running inside a container which I have already implemented in DockerClient. So using the REST API rather than doing it using `Runtime`. Also notice that this would allows you to write a test.